src/target/target_kind.cc

```diff
@@ -431,6 +431,9 @@ TVM_REGISTER_TARGET_KIND("hybrid", kDLCPU)  // line break
 
 TVM_REGISTER_TARGET_KIND("composite", kDLCPU).add_attr_option<Array<Target>>("devices");
 
+TVM_REGISTER_TARGET_KIND("test", kDLCPU)
+    .set_attr<String>("relay_to_tir", "target.test.tir_lowering");
+
```

Comment:
If anyone knows a temporary way to register this as part of the test, that'd be preferred.

Reply:
would you guys be up for adding a context manager to tvm.testing to do this? it has bitten us elsewhere. might need to add a way to delete the function, and should verify/error in such a context manager if, on deletion, it's detected that the global func was overridden

Reply:
This approach got replaced by the `Pass` approach so I think we'll leave the context manager for another PR :smile_cat: though I think it'd be sensible to have something for registering a function temporarily in Python tests.